sphinx/screens/chats/chat-common/chat-common/src/main/java/chat/sphinx/chat_common/ui/ChatFragment.kt

```diff
@@ -370,7 +370,9 @@ abstract class ChatFragment<
                                 viewModel.copyMessageText(holderState.message)
                             }
                             is MenuItemState.Delete -> {
-                                // TODO: Implement
```

Comment:
Should always prefer to utilize viewModelScope in possible, as cancellation of the lifecycleScope will cancel the suspend function being called, thus not handling the returned value.

```kotlin
// from screen
viewModel.deleteMessage(holderState.message)

// from viewmodel
fun deleteMessage(message: Message) {
    viewModelScope.launch(mainImmediate) {
        when (messageRepository.deleteMessage(message)) {
            // ......
        }
    }
}
```